go-controller/pkg/ovn/ovn.go

```diff
@@ -570,6 +584,98 @@ func (oc *Controller) WatchNetworkPolicy() error {
 	return err
 }
 
+// WatchCRD starts the watching of the CRD resource and calls back to the
+// appropriate handler logic
+func (oc *Controller) WatchCRD() error {
+	_, err := oc.watchFactory.AddCRDHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			crd := obj.(*apiextension.CustomResourceDefinition)
+			klog.Errorf("ADDING CRD - %s\n", crd.Name)
```

Comment:
@JacobTanenbaum debugging code? Shouldn't be an error in the final patch.

Reply:
done